EnglishFilter/sections/foreign.txt

```diff
@@ -9067,14 +9067,12 @@ gamme.com.tw##a[href^="https://www.gamme.com.tw/redirect.php?id="][target="_blan
 ! https://github.com/AdguardTeam/AdguardFilters/issues/15480
 @@||service.4gtv.tv/4gtv/Data/GetAD.ashx
 @@||imasdk.googleapis.com/js/sdkloader/ima3.js$domain=4gtv.tv
-||vast.doublemax.net/adserver/vast.xml$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||ads.adaptv.advertising.com/*roll&pageUrl=4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||facebook.com/*/instream/vast.xml?*&pageurl=http*.4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||pubads.g.doubleclick.net/gampad/ads?*&output=xml_vast*&url=http*.4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
+||vast.doublemax.net/adserver/vast.xml$script,xmlhttprequest,other,redirect=noopvast-2.0
```

Comment:
are `.XML` detected as script?

Reply:
I'm not sure. but as I recall, I saw it somewhere with apps, so I prefered to add it.
I think that adding `script` content type to these rules shouldn't cause any problems, as before there were `$replace` rules without any content type.

Reply:
Yep.